### DISCUSSION TITLE
Trouble with inserting values into Rhino/Grasshopper

### DISCUSSION THREAD

**Discussion by Jan-Blasiak** [philipbelesky/Caribou#13] [Q&A] [answered]
Excuse me, I'm a complete beginner to the plugin and Rhino 7, I tried following the guide on the website for importing maps, but I am unable to make it work, the Grasshopper throws me an error saying that:

 "Floating parameter file path failed to collect data"

I don't know how to properly import data, I would appriciete some help in the matter.

I'm attaching two files with the imported map, I hope it helps.

[Maps.zip](https://github.com/philipbelesky/Caribou/files/7006371/Maps.zip)

**Comment by philipbelesky** [comment]
Hi Jan! That's ok, thanks for reaching out. Could you send attach your grasshopper file or a screenshot of it? 

The map file you download from OSM needs to be brought into grasshopper by way of a 'file path' component which you right click and locate the file. That file path component then connects as per the documentation. 

I'll try to make the error message more helpful as well!

**Reply by Jan-Blasiak** [reply]
![test](https://user-images.githubusercontent.com/61629226/130074757-a1bbd970-4653-4792-b45a-f7767bb4fba0.png)
[test.gh.zip](https://github.com/philipbelesky/Caribou/files/7014596/test.gh.zip)

I can do both!

I tried directly copying what I saw on your website, but it doesn't seem to work,

Thank you for quick answer.

**Reply by Jan-Blasiak** [reply]
Err... what the error message obstructs, is the connections looking exactly like what is shown on your website.

**Reply by philipbelesky** [reply]
Strange, the file is definitely setup as it should be. Downloading the file and referencing in my own OSM file works. From what I can tell the `FilePath`component (which is a standard Grasshopper component) is having some issue with the file path provided that doesn't seem to be related to this plugin in particular. It could be that:

- The file doesn't exist in that location any more
- Rhino/Grasshopper don't have permission to read the file. Possibly this could be a weird security thing given the file seems to be on an external drive?

Other than moving the file and re-referencing it, the only other thing I can think of to try is making a `Panel` component and copy/pasting the filepath in. That should at least isolate wether the issue is with the `File Path` component or the file's path itself.

<img width="785" alt="Screen Shot 2021-08-20 at 4 38 50 pm" src="https://user-images.githubusercontent.com/495961/130191113-9d6f0422-adef-44f2-84bc-77216a339e3d.png">

**Comment by philipbelesky** [comment, marked as answer]
Hi Jan, not sure if you found a fix for this originally, but there is a new version up on the package manager that hopefully addresses this. If you had the time, I'd appreciate it if you were able to check it out and let me know how you go.

**Reply by Jan-Blasiak** [reply]
Thank you, will check as soon as I'm done with my degree, sorry for not answering before, I was experimenting with Rhino for my boss and got distracted by assignments.
